Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.8-alpine3.6
+
+ENV CGO_ENABLED=0 \
+    PATH="$PATH:/app/node_modules/.bin/"
+
+RUN apk add --no-cache bash make zip chromium yarn && \
+    yarn install && \
```

Comment:
You need to delete this line. As I mentioned earlier, we have a `make deps` now, and user will use it when actually running a container. Fetching dependencies (like `yarn install` or `dep ensure`) needs to be done at runtime, not at image build time, because they change from time to time and we don't want to ask people to recompile the image.